testing/integration-testing-hooks.ts

```diff
@@ -239,9 +239,7 @@ export class MongodSetup {
 
     let client;
     try {
-      client = await MongoClient.connect(await this.connectionString(), {
-        useNewUrlParser: true
-      } as MongoClientOptions);
+      client = await MongoClient.connect(await this.connectionString(), {});
```

Comment:
What is the issue with `useNewUrlParser: true`?

Reply:
It has been removed, there is only one URL parser now as I understand it

Reply:
Makes sense